### PR TITLE
feat(component): Impl to currentTarget

### PR DESCRIPTION
**Pull request by mixed**
ref #25

### CODE REVIEW

**Comment by mixed** [comment]
@sculove 
Could you review it?

File: test/unit/component.spec.js [hunk 1]
```diff
@@ -258,18 +258,30 @@ describe("trigger method", function() {
...
+	it("shouldn't call extended method when the Array is extends", () => {
 		//Given
 		Array.prototype.ExtendSomthing = sinon.spy();
 		oClass.on("eventType", function() {});
 		//When
 		oClass.trigger("eventType");
 		//Then
 		expect(Array.prototype.ExtendSomthing.called).to.be.false;
+	});
+	delete Array.prototype.ExtendSomthing;
```
Comment (sculove) [inline test/unit/component.spec.js]: Why did not you put the delete statement into it scope?
```js
it("shouldn't ...", () => {
   // ...
   expect(Array.prototype.ExtendSomthing.called).to.be.false;
   delete Array.prototype.ExtendSomthing;
});
```

**Comment by sculove** [comment]
update eslint 4.x and eslint-config-naver 1.0.1
and then check it.

File: test/unit/component.spec.js (hunk 1 above)
Comment (mixed) [reply test/unit/component.spec.js]: 👍  Thanks. I going to update it.

**Pull request merged**